persistence/test/test.go

```diff
@@ -22,39 +26,43 @@ type TestDBSettings struct {
 func PGTestDSN() string {
 	dsn, ok := os.LookupEnv("LNMUX_TEST_DB_DSN")
 	if !ok {
-		dsn = "postgres://bottle:bottle@localhost:45432/bottle_test?sslmode=disable"
+		dsn = "postgres://bottle:bottle@localhost:45432/template1?sslmode=disable"
 	}
 
 	return dsn
 }
 
-func ResetPGTestDB(t *testing.T, settings *TestDBSettings) (*pg.DB, string) {
+func CreatePGTestDB(t *testing.T, settings *TestDBSettings) *pg.Options {
 
+	// To create a DB, we need a connection to the server that isn't to
+	// the nonexistent DB itself. We connect to the template1 DB, which
+	// always exists.
 	if settings.DSN == "" {
```

Comment:
Pre-existing, but isn't `settings.DSN` always empty for our tests?

Reply:
Yes. I've removed the DSN and Schema fields from the test struct as well now.